include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -307,6 +307,9 @@
 #    define _ONEDPL_CPP20_RANGES_PRESENT 0
 #endif
 
+// Feature public macros
+#define ONEDPL_HAS_RANGES_API  _ONEDPL_CPP20_RANGES_PRESENT
```

Comment:
Returning back to the question with the versions: should there be different numbers (for example, oneDPL release version) for the feature macro e.g. to reflect different set of available algorithms?

Reply:
I have not an answer right now... it should be discussed within team

Reply:
Let's return to it when https://github.com/oneapi-src/oneDPL/pull/1457#discussion_r1735601257 is sorted out.

Reply:
Discussed offline with Ruslan. Agreed don't introduce any public feature macro for now. It should be worked out in the future.